docs/runbooks/upgrading-aks.md

```diff
@@ -86,3 +86,4 @@ Monitor via eg.
 ```shell
 watch -n 5 kubectl get nodes
 ```
+5. Go to `dplsh's` Dockerfile and update the `kubectl` version to match that of the upgraded AKS version
```

Comment:
Overvej at lave en runbook til hvordan man udgiver en ny version af dplsh, og link til den her. Det gør man sådan her:

1. Lav ændringerne i dplsh directory'et
2. Byg lokalt med `IMAGE_URL=dplsh IMAGE_TAG=local task build` i dplsh mappen
3. Test at det virker med `DPLSH_IMAGE=dplsh:local ./dplsh`
4. Tjek release version her: https://github.com/danskernesdigitalebibliotek/dpl-platform/releases
5. Push en branch og få review. Merge til master.
6. Push et tag af formen `dplsh-x.x.x` på master - så bygger og releaser den automatisk.

Derudover: variablen i dplsh hedder `KUBECTL_VERSION` :-)

Reply:
Test evt. processen ifbm at du skriver en runbook, det tror jeg er en god øvelse.

Reply:
Jeg har lavet det som et nyt PR :)

Reply:
The `Ordered list item prefix` error can be ignored here, c.f. https://github.com/DavidAnson/markdownlint?tab=readme-ov-file#configuration